Diffusion.xcodeproj/project.pbxproj

```diff
@@ -661,10 +675,13 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"Diffusion/Preview Content\"";
+				DEVELOPMENT_TEAM = 9CPA8P59Z6;
 				ENABLE_HARDENED_RUNTIME = YES;
 				ENABLE_PREVIEWS = YES;
 				GENERATE_INFOPLIST_FILE = YES;
 				INFOPLIST_FILE = Diffusion/Info.plist;
+				INFOPLIST_KEY_CFBundleDisplayName = Diffusers;
+				INFOPLIST_KEY_NSPhotoLibraryAddUsageDescription = "To be able to save generated images to your Photo Library, you’ll need to allow this.";
```

Comment:
I think this was already present in [`Info.plist`](https://github.com/huggingface/swift-coreml-diffusers/blob/main/Diffusion/Info.plist#L5). Did you find it necessary to add it to the project too? (Not a big deal, but it's probably better to not duplicate resources if we can).

Reply:
I'm a bit blind as to why these get added tbh. I didn't intentionally make these edits that I recall. How can I remove this change?

Reply:
Yeah, those things happen sometimes. I'll try to remove them from the PR and see if everything keeps working.